docker/rust-all.Dockerfile

```diff
@@ -67,6 +58,14 @@ EXPOSE 6186
 ENV RUST_BACKTRACE 1
 ENV RUST_LOG_FORMAT=json
 
+# add build info
+ARG GIT_TAG
+ENV GIT_TAG ${GIT_TAG}
+ARG GIT_BRANCH
+ENV GIT_BRANCH ${GIT_BRANCH}
+ARG GIT_SHA
+ENV GIT_SHA ${GIT_SHA}
```

Comment:
@geekflyer we are using `option_env` macros that will be evaluated only at build time. We should change that as well, if we want to pursue this way.